Tribler/Main/vwxGUI/GuiUtility.py

```diff
@@ -290,6 +296,9 @@ def ShowPage(self, page, *args):
             self.frame.selectedchannellist.Focus()
         elif page == 'my_files':
             self.frame.librarylist.Focus()
+        elif page == 'creditmining':
+            pass
+            # self.frame.creditmininglist.Focus()
```

Comment:
If you are not planning to do something with this commented out code, please remove it if you can.